Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "envelopers"
-version = "0.5.1"
+version = "0.6.0"
```

Comment:
I did a minor version here but it could have just been a patch - this won't change consuming the library at all.